Buildings/Experimental/DHC/CentralPlants/Cooling/Subsystems/Examples/CoolingTowerWithBypass.mo

```diff
@@ -0,0 +1,151 @@
+    show_T=true,
+    m_flow_nominal=m_flow_nominal/2,
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    dp_nominal=60000,
+    TAirInWB_nominal=273.15+25.55,
+    TWatIn_nominal=273.15+35,
+    dT_nominal=5.56,
+    PFan_nominal=4800,
+    TMin=TMin,
+    controllerType=Modelica.Blocks.Types.SimpleController.PI)
+    "Parallel cooling towers"
+    annotation (Placement(transformation(extent={{0,40},{20,60}})));
+  Buildings.BoundaryConditions.WeatherData.ReaderTMY3 weaDat(
+    final computeWetBulbTemperature=true,
+    filNam=ModelicaServices.ExternalReferences.loadResource(
+      "modelica://Buildings/Resources/weatherdata/USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.mos"))
+    annotation (Placement(transformation(extent={{-100,80},{-80,100}})));
+  Modelica.Blocks.Logical.OnOffController onOffCon(
+    bandwidth=2,
+    reference(
+      unit="K",
+      displayUnit="degC"),
+    u(
+      unit="K",
+      displayUnit="degC"))
+    "On/off controller"
+    annotation (Placement(transformation(extent={{-10,-90},{10,-70}})));
+  Modelica.Blocks.Logical.Switch swi
+    "Control switch for chilled water pump"
+    annotation (Placement(transformation(extent={{30,-90},{50,-70}})));
+  Modelica.Blocks.Sources.Constant TSwi(
+    k=273.15+14)
+    "Switch temperature for switching tower pump on"
+    annotation (Placement(transformation(extent={{-60,-96},{-40,-76}})));
+  Modelica.Blocks.Sources.Constant zer(
+    k=0)
+    "Zero flow rate"
+    annotation (Placement(transformation(extent={{-10,-120},{10,-100}})));
+  Modelica.Blocks.Sources.Constant m_flow(
+    k=m_flow_nominal)
+    "Water flow rate"
+    annotation (Placement(transformation(extent={{-10,-50},{10,-30}})));
+  Buildings.Fluid.MixingVolumes.MixingVolume vol(
+    nPorts=3,
+    redeclare final package Medium=Medium,
+    m_flow_nominal=m_flow_nominal,
+    V=0.5,
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial)
+    annotation (Placement(transformation(extent={{20,-20},{40,0}})));
+  Buildings.Fluid.Sources.Boundary_pT exp(
+    redeclare package Medium=Medium,
+    nPorts=1)
+    "Expansion vessel"
+    annotation (Placement(transformation(extent={{100,-30},{80,-10}})));
+  Modelica.Thermal.HeatTransfer.Sources.FixedHeatFlow fixHeaFlo(
+    Q_flow=0.5*m_flow_nominal*4200*5)
+    "Fixed heat flow rate"
+    annotation (Placement(transformation(extent={{-40,0},{-20,20}})));
+  Modelica.Thermal.HeatTransfer.Sensors.TemperatureSensor TVol
+    "Water temperature"
+    annotation (Placement(transformation(extent={{-60,-50},{-40,-30}})));
+  Buildings.Fluid.Movers.FlowControlled_m_flow pum(
+    redeclare final package Medium=Medium,
+    m_flow_nominal=m_flow_nominal,
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    nominalValuesDefineDefaultPressureCurve=true)
+    "Pump for chilled water loop"
+    annotation (Placement(transformation(extent={{-40,40},{-20,60}})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTCWLvg(
```

Comment:
This appears to be unused and should be deleted (Note: it will be removed if you extend `PartialStaticTwoPortCoolingTower`)